.github/workflows/test.yml

```diff
@@ -9,15 +9,28 @@ jobs:
   test:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - name: Install dependencies
-      run: npm ci
-    - name: Build
-      run: npm run build
-    - name: Check for Execution
-      uses: ./
-      with:
-        skipOnCommitMsg: "sample"
-        ghToken: ${{secrects.GH_TOKEN_3}}
-    - name: Some Job
-      run: echo "sample"
+      - name: Checkout source
+        uses: actions/checkout@v4.1.1
+      - name: Install Node (v20) 
+        id: node-install
+        uses: actions/setup-node@v4.0.2
+        with:
+          node-version: 20.11
+          cache: 'npm'
+            # allow packages inside this repo to consume from our private npm registry
```

Comment:
@vitaliibendyk check indent here, since this comment is for the next line it should start at the same indent